ansys/dpf/core/field_base.py

```diff
@@ -194,24 +210,20 @@ def scoping(self):
         >>> #The fourth elementary data of the field corresponds to 
         >>> #the element id number 586 in the mesh
         """
+        
         return self._get_scoping()
     
     @scoping.setter
     def scoping(self, scoping):
-        """
-        Parameters
-        ----------
-        scoping : Scoping
-        """
         return self._set_scoping(scoping)
                
         
     def get_entity_data(self, index):
-        """Returns the elementary data of the scoping's index in parameter
+        """Retrieves the elementary data of the scoping's index in an array.
         
         Returns
         --------
-        data : numpy.array
+        numpy.array
```

Comment:
```suggestion
        numpy.ndarray
```
`array` is actually a function.  `ndarray` is the class we're looking for.